tests/simple/simple_test.go

```diff
@@ -119,6 +136,45 @@ func TestSimpleFile(t *testing.T) {
 	if rc == nil {
 		return
 	}
+	var skipTests []string
+	firstFlag := true
+	for _, flagVal := range skipFlags {
+		fileInd := strings.Index(flagVal, "/")
+		splitFile = strings.SplitN(flagVal, "/", 2)
+		fileName := splitFile[0]
+		sectionString := splitFile[1]
+		if fileInd > 0 || sectionString == "" {
+			log.Fatal("skip_test argument must contain at least <file>/<section>, received ", flagVal)
+		}
+		for _, sectionVal := range strings.Split(sectionString, ";") {
+			sections := strings.Count(sectionVal, "/")
+			if sections == 0 {
+				if sectionVal != "" {
+					skipTests = append(skipTests, filename+"/"+sectionVal)
+				} else {
+					log.Fatal("Empty string where should be section name")
+				}
+			} else if sections == 1 {
+				lastInd := strings.LastIndex(sectionVal, "/")
+				sectionName := sectionVal[:lastInd+1]
```

Comment:
Be consistent with filename about whether or not you include the "/".  I'd prefer to keep it out and have "filename" and "sectionName" be just what they say, but including them is okay too if that's your preference, just keep in consistent.